### PR TITLE
On /download/desktop/thank-you show contribution form if no download

### LINKED ISSUE #8478: Donations links do not work
When I go to Community -> Contribute to our project, the website I land on does not seem to have an easy option to donate. In particular, I am directed to this thank you page which assumes I already donated.

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Make going to /download/desktop/thank-you without a download show contribution form

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/download/desktop/thank-you
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that if you go to the page without a download, it offers the chance to contribute

## Issue / Card

Fixes #8478

## Screenshots

image

### CODE REVIEW

File: templates/download/desktop/thank-you.html [hunk 1]
```diff
@@ -40,15 +40,14 @@ <h1 style="font-weight: 100;">Thank you for downloading Ubuntu Desktop</h1>
         </p>
         {% with version=version, system="desktop", architecture=architecture %}{% include "download/shared/_verify-checksums.html" %}{% endwith %}
         {% else %}
-        <h1>Thank you for your contribution</h1>
+        <h1>Thank you for your contributing</h1>
```
Comment (albertkol) [inline templates/download/desktop/thank-you.html:43]: Did you mean to write: "Thank you for contributing"?

**Review by albertkol — COMMENTED** [review]
Please rebase to master. It will fix the build.

File: templates/download/desktop/thank-you.html (hunk 1 above)
Comment (pmahnke) [reply templates/download/desktop/thank-you.html]: fixed...

**Review by albertkol — COMMENTED** [review]
I see a lot of CSS changes that were not there before. Have you added them by accident?

**Review by albertkol — APPROVED** [review]
:+1:

**Pull request merged**